### ISSUE TITLE
Allow overriding cache option for specific scripts

### ISSUE THREAD

**Issue by ShivamNegi** [weecology/retriever#833]
```--not-cached``` option doesn't work properly with some scripts: 

```
scripts/la_selva_trees.py:    def download(self, engine=None, debug=False):
scripts/gentry_forest_transects.py:    def download(self, engine=None, debug=False):
scripts/breed_bird_survey.py:    def download(self, engine=None, debug=False):
scripts/forest_inventory_analysis.py:    def download(self, engine=None, debug=False):
scripts/prism_climate.py:    def download(self, engine=None, debug=False):
scripts/breed_bird_survey_50stop.py:    def download(self, engine=None, debug=False):
scripts/biomass_allometry_db.py:    def download(self, engine=None, debug=False):
scripts/bioclim.py:    def download(self, engine=None, debug=False):
scripts/plant_life_hist_eu.py:    def download(self, engine=None, debug=False):
scripts/vertnet.py:    def download(self, engine=None, debug=False):
scripts/wood_density.py:    def download(self, engine=None, debug=False):
scripts/intertidal_abund_me.py:    def download(self, engine=None, debug=False):
scripts/npn.py:    def download(self, engine=None, debug=False):
```
These scripts are using their own ```download``` functions.

**Comment by ShivamNegi** [comment]
@ethanwhite , I will take this issue myself as a continuation of #134 .

**Comment by ethanwhite** [comment]
Fixed in 3ac063c850c69d016ea45fbbce917ae387065546. Thanks @ShivamNegi!
